### DISCUSSION TITLE
Github Fork -  como funciona?

### DISCUSSION THREAD

**Discussion by sebasvelasco353** [sebasvelasco353/FDP-2022_01#2] [Announcements]
Les dejo un video donde [Daniel Shiffman](https://youtu.be/_NrSWLQsDL4) explica muy bien y rapido como funcionan los fork en github, recuerden que esta va a ser la forma en la que les comparto el contenido de las clases y en la que uds haran entrega de los talleres y ejercicios de las clases.

denle una revisada a [este video](https://youtu.be/_NrSWLQsDL4) y si tienen alguna duda pueden preguntarme en la clase o como siempre, hacer preguntas en la seccion de discusiones del repo.

tambien les dejo [este otro](https://youtu.be/f90683yjtyE) en el que explican como mantener su repo al dia con el repo principal de la clase, esto deben hacerlo siempre despues de cada clase para tener los ultimos cambios agregados.

**Comment by Pipelenisf** [comment]
Hola Sebas, dos preguntas:
- ¿Es necesario crear la SSH Key para mantener el repo al dia?
- Cuando pongo el comando git merge upstream/main solo me aparece "Already up to date" ¿Me salté algún paso?
En conclusión no he podido agregar lo que subiste el lunes a mí repositorio

**Reply by AlejandroRv19** [reply]
Hola Profe Soy Alejandro rueda y no sé porque el sidebar no me funciona me ayudas porfa

https://drive.google.com/file/d/1QQfcNpVvaAdCQwWfm2HvEXxvMuLaDHaL/view?usp=drivesdk

**Reply by sebasvelasco353** [reply]
@Pipelenisf puedes eliminar el upstream con `git remote rm upstream` y volver a agregar upstream con la url https://github.com/sebasvelasco353/FDP-2022_01.git

**Reply by sebasvelasco353** [reply]
> Hola Profe Soy Alejandro rueda y no sé porque el sidebar no me funciona me ayudas porfa
> 
> 
> 
> https://drive.google.com/file/d/1QQfcNpVvaAdCQwWfm2HvEXxvMuLaDHaL/view?usp=drivesdk

Hola Alejandro, acuérdate que si deseas usar grid como la forma de crear tu layout debes darle la propiedad `display: grid` al contenedor.

Una vez hagas esto los elementos se posicionarán dependiendo de las columnas y filas que les asignes dentro de ese contenedor.
